### DISCUSSION TITLE
Extend readOnlyRootFileSystem to include NAP WAF when present

### DISCUSSION THREAD

**Discussion by brianehlert** [nginx/kubernetes-ingress#5156] [Ideas]
Customers use the readOnlyRootFileSystem capability to align with security policy and customers would like to also use this when NAP WAF is included with NIC.

The current implementation of readOnlyRootFileSystem does not support the NAP WAF module and thus the capability needs to be extended to support NAP WAF module behavior and paths necessary.

**Comment by vepatel** [comment]
Issue: https://github.com/nginxinc/kubernetes-ingress/issues/5291

**Comment by brianehlert** [comment]
Given the new support for WAF v5 with NIC 3.6, is this still relative?

**Comment by AlexFenlon** [comment]
Closing as this issue has been resolved with WAF v5 [here](https://github.com/nginx/kubernetes-ingress/issues/5291?issue=nginx%7Ckubernetes-ingress%7C6592)
